src/env_vars.py

```diff
@@ -26,7 +27,7 @@ def get_env_var(var_name, default=None):
 IP_RANGE = get_env_var("IP_RANGE", "172.18.0.0/24")
 GATEWAY = get_env_var("GATEWAY", "172.18.0.1")
 RUNNING_IN_CI = get_env_var("CI")
-API_REQUEST_TIMEOUT = get_env_var("API_REQUEST_TIMEOUT", 20)
+API_REQUEST_TIMEOUT = get_env_var("API_REQUEST_TIMEOUT", 50)
```

Comment:
Sometimes I got error that REST isn't ready or connection timeout this could be my machine but I added it just to avoid it happening on CI but I can remove it

Reply:
Done